main.go

```diff
@@ -1,12 +1,28 @@
 package main
 
 import (
+	"flag"
+	"log"
+
 	"github.com/hashicorp/terraform/plugin"
+	"github.com/hashicorp/terraform/terraform"
 	"github.com/sourcegraph/terraform-provider-site24x7/site24x7"
+	"github.com/sourcegraph/terraform-provider-site24x7/site24x7/oauth"
 )
 
+var oauthFile = flag.String("oauth_file", "", "path to the oauth_file")
+
 func main() {
+	flag.Parse()
+
+	ator, err := oauth.NewAuthenticator(*oauthFile)
```

Comment:
`ator` feels like made by some non-native speaker (i.e. me) would come up with 😂 I used `iset` for `intersection` in another PR...

Reply:
it's short but still distinct